.github/workflows/build.yml

```diff
@@ -270,6 +270,7 @@ jobs:
         submodules: true
     - name: Bootstrap
       run: |
+        rm -f '/usr/local/bin/2to3'
```

Comment:
```suggestion
        rm -f '/usr/local/bin/2to3' || true
```

Reply:
`rm -f` always returns zero even the file doesn't exist.